tldw_Server_API/app/core/Writing/manuscript_analysis.py

```
@@ -0,0 +1,178 @@
+        kwargs["model"] = model
+
+    content: str = ""
+    try:
+        response = await perform_chat_api_call_async(**kwargs)
+        content = _extract_content(response)
+        content = _strip_markdown_fences(content)
+        return json.loads(content)
+    except json.JSONDecodeError:
+        logger.warning("Failed to parse analysis JSON response")
+        return {"error": "Failed to parse LLM response", "raw": content[:500] if content else ""}
+    except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)